src/Sarif/Sarif.ruleset

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Rules for Sarif" Description="Code analysis rules for Sarif.csproj." ToolsVersion="14.0">
+  <IncludeAll Action="Warning" />
+  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
+    <Rule Id="CA2104" Action="None" />
+  </Rules>
+</RuleSet>
```

Comment:
I see that you created a rule set, but the rule set is not mentioned in the project file Sarif.csproj (that is, I don't see any changes to Sarif.csproj).

You _could_ make the required changes by hand (and I could show you how), but I'm curious why it didn't happen automatically when you created the rules file. Did you create the rules file by going to the Sarif project properties page in VS, selecting the Code Analysis tab, clicking Open, and then unchecking the check box for this rule?

+ @michaelcfanning